### PR TITLE
feat: Python example for api-sqs-lambda

### PR DESCRIPTION
**Pull request by akshitkh**
Added a Python example for an API-SQS-Lambda integration that creates the following architecture:

1. API Gateway will a POST method
2. SQS Queue
3. API Gateway Integration with the SQS Queue
4. Lambda function triggered by the SQS queue

Requests to the API are enqueued in SQS, and processed by the Lambda function.

Fixes # 

---

By submitting this pull request, I confirm that my contribution is made under the terms of the [Apache 2.0 license].

[Apache 2.0 license]: https://www.apache.org/licenses/LICENSE-2.0

### CODE REVIEW

File: python/api-sqs-lambda/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+
+# API Gateway + SQS + Lambda
+
+Creates an API Gateway API with a POST Method, a SQS queue, and a Lambda function. Requests to the API are enqueued into the SQS queue, which triggers the Lambda function.
+
+![Architecture](architecture.png)
+
+The cdk.json file tells the CDK Toolkit how to execute your app.
```
Comment (nija-at) [inline python/api-sqs-lambda/README.md:8]:
```suggestion
The `cdk.json` file tells the CDK Toolkit how to execute your app.
```

File: python/api-sqs-lambda/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+This project is set up like a standard Python project. The initialization process also creates a virtualenv within this project, stored under the .env directory. To create the virtualenv it assumes that there is a python3 (or python for Windows) executable in your path with access to the venv package. If for any reason the automatic creation of the virtualenv fails, you can create the virtualenv manually.
```
Comment (nija-at) [inline python/api-sqs-lambda/README.md:10]: Added some backquotes and line wrap to 120.

```suggestion
This project is set up like a standard Python project. The initialization process also creates a virtualenv within this
project, stored under the `.env` directory. To create the virtualenv it assumes that there is a `python3` (or `python`
for Windows) executable in your path with access to the `venv` package. If for any reason the automatic creation of the
virtualenv fails, you can create the virtualenv manually.
```

File: python/api-sqs-lambda/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+
+# API Gateway + SQS + Lambda
```
Comment (nija-at) [inline python/api-sqs-lambda/README.md:2]: Can you add a section at the end of this file, letting people know how they can interact with the APIGateway after the stack has been deployed and how to check the message was received by the Lambda function and view the contents of the message via the function's logs?

You could provide a sequence of AWS CLI commends with 1 line commentary on what each command is doing, similar to the steps you've used to describe the steps to activate virtualenv and `cdk synth`.

Comment (akshitkh) [reply python/api-sqs-lambda/README.md:2]: Added testing instructions in the README file. Hope this helps!

File: python/api-sqs-lambda/README.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+```
+$ pip install -r requirements.txt
+```
+
+At this point you can now synthesize the CloudFormation template for this code.
+
+```
+$ cdk synth
+```
+
```
Comment (nija-at) [inline python/api-sqs-lambda/README.md:45]:
```suggestion

## Testing the app

```

**Pull request merged**